.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+A short description of the changes in this PR.
+
+## Jira Issue ID
+
```

Comment:
```suggestion
[DAS-XXX](https://bugs.earthdata.nasa.gov/browse/DAS-XXX)
```
What about that, make it a little easier?

Reply:
[Sounds good](https://github.com/nasa/harmony-browse-image-generator/pull/1/commits/2262f4ff074d985d3db397f045f462c9cecd1ba9)